tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -224,7 +264,14 @@ def test_disruptive_during_pod_pvc_deletion_and_io(
 
         # Start IO on IO pods
         for pod_obj in io_pods:
-            pod_obj.run_io(storage_type='fs', size=f'{self.pvc_size - 1}G')
+            if pod_obj.pvc.access_mode == constants.ACCESS_MODE_RWX:
```

Comment:
these lines are used a few times - please create a function in this file to cover this implementation

Reply:
Done